scripts/image-names.sh

```diff
@@ -5,6 +5,15 @@
 
 # Be consistent about naming our images. This script helps us understand inconsistencies.
 
+gradleFolder() {
+  case "$1" in
+    pdfgenerator|featuretoggle|assessclaim*) echo "./domain-rrd/service-python/$1";;
+    cc-app) echo "domain-cc/$1";;
+    ee-max-cfi-app) echo "domain-ee/$1";;
+    *) echo "./$1";;
+  esac
+}
+
```

Comment:
Remove this
```suggestion
```

Reply:
Why are we removing this section? can you please explain?

Reply:
It's no longer used anywhere